.github/workflows/inspektor-gadget.yml

```diff
@@ -368,7 +368,7 @@ jobs:
           exit-code: 1
```

Comment:
nit: (not related to this PR) I think we can remove the steps from [L350-L357](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/.github/workflows/inspektor-gadget.yml#L350-L357), I might have added them in copy-paste but seems like they aren't really needed.